atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -100,6 +102,7 @@ private AtlasDbHttpClients() {
      */
     private static Client newOkHttpClient(Optional<SSLSocketFactory> sslSocketFactory) {
         com.squareup.okhttp.OkHttpClient client = new com.squareup.okhttp.OkHttpClient();
+        client.setConnectionPool(new ConnectionPool(100, TimeUnit.MILLISECONDS.convert(10, TimeUnit.MINUTES)));
```

Comment:
Added constants. However it doesn't seem like Client really exposes anything to verify this (can't even get the ConnectionPool since OkHttpClient wraps this).